pyproject.toml

```diff
@@ -2,11 +2,10 @@
 addopts = "-q"
 filterwarnings = [
     "error", # Fail the tests if there are any warnings.
-    "ignore:^find_module\\(\\) is deprecated and slated for removal in Python 3.12; use find_spec\\(\\) instead$:DeprecationWarning:importlib",
```

Comment:
This can be removed from the coockiecutter once we are on the new version in all proejcts. Added a remainder about this.